github/data_source_github_repository_branch.go

```diff
@@ -0,0 +1,63 @@
+package github
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGithubRepositoryBranch() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGithubRepositoryBranchRead,
+
+		Schema: map[string]*schema.Schema{
+			"repository": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"branch": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "master",
+			},
+
```

Comment:
Just pulling an example at random, there are other resources that include a blank line between computed and non-computed attributes. e.g. [github_repository](https://github.com/terraform-providers/terraform-provider-github/blob/c533ed9cbc2c17887d150cafcea0509bfcd78b08/github/resource_github_repository.go#L112).

Reply:
Yea i see that now.. it would be nice if there was some standard because it feels random at the moment (e.g. why separate computed fields but not required or optional).

Please ignore this comment and any others relating to removing lines between attribute names then 👍